src/Microsoft.IdentityModel.Tokens/BaseConfigurationManager.cs

```diff
@@ -20,6 +22,10 @@ public abstract class BaseConfigurationManager
         private BaseConfiguration _lastKnownGoodConfiguration;
         private DateTime? _lastKnownGoodConfigFirstUse = null;
 
+        private EventBasedLRUCache<BaseConfiguration, DateTime> _lkgConfigurationCache = null;
```

Comment:
be consistent with naming.
_lkgConfigurationCache vs. _lastKnownGoodConfigFirstUse vs. _lastKnownGoodConfiguration vs. _lastKnonwGoodConfigurationSizeLimit

_lkgConfiguration is my choice.

Reply:
Going to use _lastKnownGoodConfigurationCache and it is consistent with other public property.